src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] machines = {truck, bulldozer, excavator};
```

Comment:
```suggestion
        Machine[] machines = {new Truck(), new Bulldozer(), new Excavator()};
```
we can shorten it even more, if we don't store them in local variables